_data/form.yml

```diff
@@ -0,0 +1,831 @@
+
+- title: Worker's compensation insurance
+  fields:
+  - label: What best describes your worker's compensation insurance coverage for this project?
+    type: radio
+    options:
+    - label: "I am self-insured and have a certificate of consent."
+      hides: comp-II
+      shows: comp-I
+
+    - label: "I have third-party insurance."
+      shows: comp-II
+
+    - label: "I plan to hire a contractor who will follow worker's compensation laws."
+      hides: comp-II
+      shows: comp-V
+
+    - label: "I don't need insurance because I'm not hiring anyone."
+      hides: comp-II
+      shows: comp-IV
+
+    - label: "I don't need insurance because the work will cost under $100."
+      hides: comp-II
+      shows: comp-III
+
+  - group: comp-II
+    fields:
+    - label: Name of insurance carrier
+
+    - label: Policy number
+      type: number
+
+- title: Seismic work
+  fields:
+  - label: >
+      Have you filed for mandatory seismic retrofitting under [Chapter 4D of the Existing Building Code](http://library.amlegal.com/nxt/gateway.dll/California/sfbuilding/existingbuildingcode2016edition/chapter4dmandatoryearthquakeretrofitofwo?f=templates$fn=default.htm$3.0)?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: seismic-retro-mandatory
+      hides: seismic-retro-number
+
+    - label: "No"
+      shows: seismic-retro-mandatory
+      hides: seismic-retro-number
+
+  - group: seismic-retro-number
+    fields:
+    - label: Enter your seismic retrofitting permit application number.
+      type: number
+
+  - label: >
+      Have you filed for voluntary seismic strengthening under [Administrative Bulletin AB-094](https://sfdbi.org/sites/default/files/migrated/FileCenter/Documents/Administrative_Bulletins/2013_AB/AB_094_updated_010114.pdf)?
+    type: radio
+    options:
+    - label: "Yes"
+      shows: seismic-strength
+
+    - label: "No"
+      hides: seismic-strength
+
+  - group: seismic-strength
+    fields:
+    - label: Enter your seismic strengthening permit application number.
+      type: number
+
+  - label: >
+      Does the building comply with the ordinances for ADUS: [95-17](http://default.sfplanning.org/plans-and-programs/planning-for-the-city/accessory-dwelling-units/ADU_Ord_095-17_170125_signed.pdf), [162-16](https://sfgov.legistar.com/View.ashx?M=F&ID=4617538&GUID=2625970B-6704-46C2-A972-68BF0C206D9C) or [162-17](https://sfbos.org/sites/default/files/o0162-17.pdf)?
+    helptext: This will be subject to review from the Planning department.
+    type: radio
+    options:
+    - label: "Yes"
+    - label: "No"
+
+- title: Environmental Assessment
+  fields:
+
+  - label: |
+      If the site of your planned ADU has:
+
+      - A current or former gas station
+
+      - A parking lot
+
+      - Auto repair
+
+      - Dry cleaners
+
+      - Heavy manufacturing use
+
+      - Underground storage tanks
+
+      You need to upload an environmental site assessment.
```

Comment:
Persis Separate this into two questions...

- Does your site have X?
- [if yes] Great! Upload your assessment

Reply:
Scott says: explore switching this to checkboxes again